src/lib/commonPropTypes.ts

```diff
@@ -6,11 +6,39 @@ export interface CreateCommonConfig {
   children?: boolean | 'node' | 'element'
   as?: boolean
   className?: boolean
-  color?: boolean
+  color?: boolean | 'single' | 'complex'
```

Comment:
~~`single`~~ -> `simple` might be better in conjunction with `complex` what do you think?

Reply:
Agreed, it should either be `single | multiple` or `simple | complex` (I like more the second combination)